<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1107</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: gray;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="ecgWaveGraph" native="true">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>281</width>
      <height>241</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;</string>
    </property>
    <widget class="QProgressBar" name="cprBar">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>60</y>
       <width>31</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">selection-background-color: black;
background-color: lightblue;</string>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="textDirection">
      <enum>QProgressBar::TopToBottom</enum>
     </property>
    </widget>
    <widget class="QFrame" name="statusBarQFrame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QProgressBar" name="batteryLevelBar">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>0</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu Mono</family>
        <pointsize>7</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">selection-background-color: green;
background-color: black;</string>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="format">
       <string>%p%</string>
      </property>
     </widget>
     <widget class="QLabel" name="electrodeLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Electrode not connected</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="shockButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>380</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton {
    background-color: gray;
	border-radius:45px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="project.qrc">
      <normaloff>:/pic/shockButton.svg</normaloff>
      <normalon>:/pic/shockButton_on.svg</normalon>:/pic/shockButton.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>150</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>0</y>
      <width>521</width>
      <height>631</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="styleSheet">
        <string notr="true">background-color: lightgray;</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Test Panel</string>
        </attribute>
        <widget class="QWidget" name="testPanel" native="true">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>521</width>
           <height>601</height>
          </rect>
         </property>
         <widget class="QWidget" name="gridLayoutWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>230</y>
            <width>521</width>
            <height>371</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QPushButton" name="tachycardiaButton">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>100</height>
              </size>
             </property>
             <property name="tabletTracking">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton:hover:!pressed
{
  border: 1px solid red;
}</string>
             </property>
             <property name="text">
              <string>Simulate Tachycardia</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="fibrillationButton">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>100</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton:hover:!pressed
{
  border: 1px solid red;
}</string>
             </property>
             <property name="text">
              <string>Simulate Fibrillation</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="otherRhythmsButton">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>100</height>
              </size>
             </property>
             <property name="tabletTracking">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton:hover:!pressed
{
  border: 1px solid red;
}</string>
             </property>
             <property name="text">
              <string>Simulate Other Rhythms</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="deadPatientButton">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>100</height>
              </size>
             </property>
             <property name="tabletTracking">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton:hover:!pressed
{
  border: 1px solid red;
}</string>
             </property>
             <property name="text">
              <string>Simulate Dead Patient</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>210</y>
            <width>131</width>
            <height>61</height>
           </rect>
          </property>
          <property name="text">
           <string>Simulate Scenarios</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
         </widget>
         <widget class="QWidget" name="gridLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>511</width>
            <height>231</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="leftMargin">
            <number>2</number>
           </property>
           <property name="topMargin">
            <number>2</number>
           </property>
           <property name="rightMargin">
            <number>2</number>
           </property>
           <property name="bottomMargin">
            <number>2</number>
           </property>
           <property name="spacing">
            <number>5</number>
           </property>
           <item row="4" column="0">
            <widget class="QLabel" name="testPanelLabel_5">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>New patient</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="testPanelLabel_1">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>Change remaining battery</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QComboBox" name="changeAgeComboBox">
             <property name="minimumSize">
              <size>
               <width>82</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>82</width>
               <height>30</height>
              </size>
             </property>
             <property name="currentText">
              <string>Adult</string>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <item>
              <property name="text">
               <string>Adult</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Child</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="fillBatteryButton">
             <property name="text">
              <string>Fill Battery</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="testPanelLabel_2">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>Change age group</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="testPanelLabel_3">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>Pad placed</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QCheckBox" name="electrodeConnectedCheckBox">
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QCheckBox::indicator{
	width: 20px;
	height: 20px;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QDoubleSpinBox" name="batterySpinBox">
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximum">
              <double>100.000000000000000</double>
             </property>
             <property name="value">
              <double>100.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="testPanelLabel_4">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>Electrode connected</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QCheckBox" name="padPlacedCheckBox">
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QCheckBox::indicator{
	width: 20px;
	height: 20px;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QPushButton" name="createPatientButton">
             <property name="text">
              <string>CreatNew</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="testPanelLabel_6">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>Touch patient</string>
             </property>
            </widget>
           </item>
           <item row="5" column="2">
            <widget class="QCheckBox" name="patientTouchedCheckBox">
             <property name="styleSheet">
              <string notr="true">QCheckBox::indicator{
	width: 20px;
	height: 20px;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="padSelectionComboBox">
             <property name="minimumSize">
              <size>
               <width>82</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>90</width>
               <height>30</height>
              </size>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <item>
              <property name="text">
               <string>AdultPad</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>ChildPad</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>Patient Info</string>
        </attribute>
        <widget class="QWidget" name="gridLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>521</width>
           <height>171</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Age group</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Heart rate</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Heart condition</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="patientInfoAge">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>unknown</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="patientInfoHeartRate">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>unknown</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="patientInfoCondition">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>unknown</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>610</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Electrode Pad Indicator</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>480</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>ShockButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="powerOnButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>480</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>141</width>
      <height>121</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>141</width>
      <height>121</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="project.qrc">
      <normaloff>:/pic/powerOff.svg</normaloff>
      <normalon>:/pic/powerOn.svg</normalon>:/pic/powerOff.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>120</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="step1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="project.qrc">
      <normaloff>:/pic/step1.svg</normaloff>
      <normalon>:/pic/step1_light.svg</normalon>
      <disabledoff>:/pic/step1.svg</disabledoff>
      <disabledon>:/pic/step1_light.svg</disabledon>:/pic/step1.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>230</width>
      <height>110</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="step2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="project.qrc">
      <normaloff>:/pic/step2.svg</normaloff>
      <normalon>:/pic/step2_light.svg</normalon>
      <disabledoff>:/pic/step2.svg</disabledoff>
      <disabledon>:/pic/step2_light.svg</disabledon>:/pic/step2.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>230</width>
      <height>110</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="step3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>170</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="project.qrc">
      <normaloff>:/pic/step3.svg</normaloff>
      <disabledoff>:/pic/step3.svg</disabledoff>
      <disabledon>:/pic/step3_light.svg</disabledon>:/pic/step3.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>230</width>
      <height>110</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="step4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>340</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="project.qrc">
      <normaloff>:/pic/step4.svg</normaloff>
      <disabledoff>:/pic/step4.svg</disabledoff>
      <disabledon>:/pic/step4_light.svg</disabledon>:/pic/step4.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>230</width>
      <height>110</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="step5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>181</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: gray;
border-radius: 20ox;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="project.qrc">
      <normaloff>:/pic/step5.svg</normaloff>
      <disabledoff>:/pic/step5.svg</disabledoff>
      <disabledon>:/pic/step5_light.svg</disabledon>:/pic/step5.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>320</width>
      <height>140</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="padLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>530</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="project.qrc">:/pic/electrodePadOff.svg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QOpenGLWidget" name="blackScreen">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>281</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1107</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="project.qrc"/>
 </resources>
 <connections/>
</ui>
